src/Sulu/Bundle/SecurityBundle/UserManager/UserManager.php

```diff
@@ -36,18 +36,14 @@
 use Sulu\Component\Security\Authentication\UserInterface;
 use Sulu\Component\Security\Authentication\UserRepositoryInterface;
 use Symfony\Component\PasswordHasher\Hasher\PasswordHasherFactoryInterface;
-use Symfony\Component\Security\Core\Encoder\EncoderFactoryInterface;
 
 class UserManager implements UserManagerInterface
 {
     use RelationTrait;
 
-    /**
-     * @param PasswordHasherFactoryInterface|EncoderFactoryInterface|null $passwordHasherFactory
-     */
     public function __construct(
         protected ObjectManager $em,
-        private $passwordHasherFactory,
+        private ?PasswordHasherFactoryInterface $passwordHasherFactory,
```

Comment:
It should not longer be the case that the hasher encoder can be null. So I think we can remove nullability here.

```suggestion
        private PasswordHasherFactoryInterface $passwordHasherFactory,
```